benchexec/tools/ultimate.py

```diff
@@ -27,14 +27,49 @@ class UltimateTool(benchexec.tools.template.BaseTool):
     Abstract tool info for Ultimate-based tools.
     """
 
+    REQUIRED_PATHS = [
+              "artifacts.xml",
+              "config",
+              "configuration",
+              "data",
+              "features",
+              "p2",
+              "plugins",
+              "LICENSE",
+              "LICENSE.GPL",
+              "LICENSE.GPL.LESSER",
+              "README",
+              "Ultimate",
+              "Ultimate.ini",
+              "Ultimate.py",
+              "z3",
+              "mathsat",
+              "cvc4",
+              ]
+
     def executable(self):
         return util.find_executable('Ultimate.py')
 
     def version(self, executable):
         return self._version_from_tool(executable)
 
     def cmdline(self, executable, options, tasks, spec, rlimits):
-        return [executable] + [spec] + options + ['--full-output'] + tasks
+        if executable == None: 
+            raise Exception('No executable specified')
+        
+        cmdline = [executable]
+        
+        if spec != None: 
+            cmdline = cmdline + ['--spec'] + [spec]
+            
+        if tasks:
+            cmdline = cmdline + ['--file'] + tasks            
```

Comment:
Is this really the intended way if the are multiple files in `tasks`?
If Ultimate does not support multiple files, it would be good to `assert` this.